gpiozero/tones.py

```diff
@@ -53,7 +53,15 @@ class to easily represent musical tones. The class can be constructed in a
 
     Several attributes are provided to permit conversion to any of the
     supported construction formats: :attr:`frequency`, :attr:`midi`, and
-    :attr:`note`.
+    :attr:`note`. Methods are provided to step :meth:`up` or :meth:`down` to
+    adjacent MIDI notes.
```

Comment:
Oh, interesting. Probably worth adding this much-clearer explanation to the 'up` and `down` doscstrings too, as IMHO (as a non-musician) talking about moving up or down adjacent MIDI notes is much easier to understand than moving a specific number of semi-tones above/below a frequency...?